pyaedt/modeler/advanced_cad/stackup_3d.py

```diff
@@ -1507,7 +1524,11 @@ def add_layer(
         name : str
             Layer name.
         layer_type : str, optional
-            Layer type. Options are ``"S"``, ``"D"``, and ``"G"``. The default is ``"S"``.
+            Layer type. Options are
```

Comment:
```suggestion
            Layer type. The default is ``"S"``. Options are:
  
```
A blank line is needed before the first item in a list to render properly in the doc. We put the default before the options.